src/main/java/com/epam/tc/dto/ListDto.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.tc.dto;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Accessors(chain = true)
+public class ListDto {
```

Comment:
The same comment about annotations as in `LabelDto.java`

Reply:
Fix this